src/main/java/core/basesyntax/service/impl/DataParserImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+    private static final String SEPARATOR = ",";
+    private static final int INFORMATION_LINE_INDEX = 0;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parse(List<String> lines) {
+        lines.remove(INFORMATION_LINE_INDEX);
+        List<FruitTransaction> transactions = new ArrayList<>();
+        lines = removeSpaces(lines);
+        for (String line : lines) {
+            transactions.add(parseFruitTransaction(line));
+        }
+        return transactions;
+    }
+
+    private FruitTransaction parseFruitTransaction(String line) {

```

Comment:
The method 'parseFruitTransaction' is private and tightly coupled with the 'parse' method. This is fine for now, but if you plan to extend functionality or reuse this method elsewhere, consider making it public and moving it to a separate utility class.